cmd/libsnap-confine-private/cgroup-pids-support.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "cgroup-pids-support.h"
+
+#include "cgroup-support.h"
+
+static const char *pids_cgroup_dir = "/sys/fs/cgroup/pids";
+
+void sc_cgroup_pids_join(const char *snap_security_tag, pid_t pid) {
```

Comment:
The sc_cgroup_pids_join() takes a snap_security tag but sc_croup_freezer_join() takes a snap_name. Might be nice to unify this (unless there is a good reason for this to be different of course).

Reply:
It is different for a reason. We may be able to change this in v2 with a real hierarchy where there's a snapd hierarchy, populated by snap names and finally by security tags (apps and hooks) where actual processes reside.